test/e2e/storage/volumeattributesclass.go

```diff
@@ -171,4 +184,141 @@ var _ = utils.SIGDescribe("VolumeAttributesClass", feature.VolumeAttributesClass
 		}))
 		framework.ExpectNoError(err, "timeout while waiting to confirm VolumeAttributesClass %q deletion", updatedVolumeAttributesClass.Name)
 	})
+
+	framework.It("should be protected by vac-protection finalizer", func(ctx context.Context) {
+		vacClient := f.ClientSet.StorageV1beta1().VolumeAttributesClasses()
+
+		initialVAC := &storagev1beta1.VolumeAttributesClass{
+			TypeMeta: metav1.TypeMeta{
+				Kind: "VolumeAttributesClass",
+			},
+			ObjectMeta: metav1.ObjectMeta{
+				GenerateName: "e2e-",
+			},
+			DriverName: "e2e-fake-csi-driver",
+			Parameters: map[string]string{
+				"foo": "bar",
+			},
+		}
+
+		ginkgo.By("Creating a VolumeAttributesClass")
+		createdVAC, err := vacClient.Create(ctx, initialVAC, metav1.CreateOptions{})
+		framework.ExpectNoError(err, "Failed to create the requested VolumeAttributesClass")
+		ginkgo.DeferCleanup(framework.IgnoreNotFound(vacClient.Delete), createdVAC.Name, metav1.DeleteOptions{})
+
+		ginkgo.By("Verifying that the vac-protection finalizer is set")
+		err = waitForVolumeAttributesClassFinalizerSet(ctx, f.ClientSet, createdVAC.Name, framework.Poll, VacUpdatingTimeout)
+		framework.ExpectNoError(err)
+
```

Comment:
please add a check for the createdPVC. It doesn't contain the given finalizer.

Reply:
sure, let me add it.